cpython-windows/build.py

```diff
@@ -1869,16 +1869,16 @@ def find_additional_dependencies(project: pathlib.Path):
         {"name": "shlwapi", "system": True},
         {"name": "version", "system": True},
         {"name": "ws2_32", "system": True},
-        # pathcch is Windows 8+ only. Python 3.9 dropped support for Windows 7.
-        # So this dependency is technically incorrect on Python 3.8.
-        {"name": "pathcch", "system": True},
         # In addition to the ones explicitly in the project, there are some
         # implicit link libraries not present. We list those as well.
         {"name": "Ole32", "system": True},
         {"name": "OleAut32", "system": True},
         {"name": "User32", "system": True},
     ]
 
+    if python_majmin in ["39", "310"]:
```

Comment:
Not sure how to make this more future proof, it would be nice to say "if greater than Python 3.9"

Reply:
We're not supporting any versions older than 3.8. So changing this to `if python_majmin != "38":` should do the job. I'll change this in flight.